### ISSUE TITLE
Training process freezes in the very beginning of the first epoch

### ISSUE THREAD

**Issue by EtagiBI** [fizyr/keras-retinanet#1539]
Hello, I'm trying to implement Keras RetinaNet form this repository for my object detection task.
For some reason the training process freezes in the very beginning of the first epoch:
![изображение](https://user-images.githubusercontent.com/20855725/120665957-a6156500-c4a5-11eb-9e0e-6e679e1f3741.png)

Here's the command I use for training:
`python keras_retinanet/bin/train.py --batch-size 4 --epochs 100 --steps 694 --lr 0.00001 --compute-val-loss --no-resize --reduce-lr-patience 10 coco E:\keras-retinanet-master\keras_retinanet\MY`

Am I doing something wrong?

**Comment by hracham** [comment]
for me didn't work too

**Comment by mrqingrongfa** [comment]
Hello! I am facing the same issue! Have you guys resolved the issue? :(
